books/RayTracingTheNextWeek.html

```diff
@@ -94,10 +93,35 @@
 </div>
 
 
+Managing Time
+--------------
+Before continuing, let's think about time, and how we might manage it across one or more renders.
+There are two aspects of shutter timing to think about: the time from one shutter opening to the
+next shutter opening, and how long the shutter stays open for each frame. Standard movie film used
+to be shot at 24 frames per second. Modern digital films can be 30 or even 60 frames per second.
```

Comment:
"Standard movie film used to be shot at 24 frames per second"

Do you mean
1. Film - the photochemical sandwich on plastic substrate
2. Film - A movie

Because you use
"Modern digital films" to mean "modern digital movies"

You might be using the two different meanings of "film" across these two sentences -> confusing.

Reply:
Changed to keep "film" a synonym for celluloid. Used movie for the composite, digital movie for digital only.